v5/core/utils.go

```diff
@@ -268,26 +268,47 @@ func SliceContains(slice []string, contains string) bool {
 
 // GetQueryParam() returns a pointer to the value of query parameter `param` from urlStr,
 // or nil if not found.
-func GetQueryParam(urlStr *string, param string) (*string, error) {
+func GetQueryParam(urlStr *string, param string) (value *string, err error) {
 	if urlStr == nil || *urlStr == "" {
-		return nil, nil
+		return
 	}
 
-	u, err := url.Parse(*urlStr)
+	urlObj, err := url.Parse(*urlStr)
 	if err != nil {
-		return nil, err
+		return
 	}
 
-	q, err := url.ParseQuery(u.RawQuery)
+	query, err := url.ParseQuery(urlObj.RawQuery)
 	if err != nil {
-		return nil, err
+		return
 	}
 
-	v := q.Get(param)
+	v := query.Get(param)
 	if v == "" {
-		return nil, nil
+		return
+	}
+
+	value = &v
+
+	return
+}
+
+// GetQueryParamAsInt() returns a pointer to the value of query parameter `param` from urlStr
```

Comment:
By default the documentation strings should only contain the name without parentheses, but you can leave it as is. :)
```suggestion
// GetQueryParamAsInt returns a pointer to the value of query parameter `param` from urlStr
```

Reply:
As I see the `GetQueryParam` function above has the same "issue".

Reply:
done